examples/basics/plotting/grid_uneven_col.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) 2015, Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""
+columns ->
+  0     1    2
+0 +-----+----+------+
+  | 1   |  2 |  2   |
+1 +-----+----+------+
+  |   3 |  3 |   4  |
+2 +-----+----+------+
```

Comment:
Ah yeah, I should cleanup the docstrings. Will do. I have to center the labels as well